chef/cookbooks/crowbar/templates/default/crowbar-upgrade-os.sh.erb

```diff
@@ -76,13 +76,28 @@ initiate_node_upgrade()
       sed -i s/django.utils.log.NullHandler/logging.NullHandler/g $horizon_custom_config
     fi
 
+    # SOC-11203: record old setting and disable lsof check for running
+    # processes since it may hang indefinitely
+
+    zypper --non-interactive install crudini
+    check_access_old=$(crudini --get /etc/zypp/zypper.conf commit psCheckAccessDeleted || true)
```

Comment:
What's the reason for `|| true` ? We're not checking the return value, and if the key is not there, it won't change anything...

Reply:
I thought we had a `set -e` in there...but I just checked and we don't. So theoretically it's redundant. Actually I'm currently pondering another overhaul in the light of Jacek's "hopefully it won't die before we restore the old version" comment: a backup file instead would preserve the state we might lose and render that and a lot more crudini mongering redundant. So if you'd like to see it gone I'd remove it in the course of that.